apps/artwork/components/related_artworks/index.jade

```diff
@@ -17,6 +17,13 @@ if artwork.layer && artwork.layer.artworks && artwork.layer.artworks.length
           )
             = layer.name
 
+        if artwork.related.length && user && user.hasLabFeature('Others also viewed')
```

Comment:
if `artwork.related` is null might this crash?

`if artwork.related?.length && user?.hasLabFeature('Others also viewed')`  i might be wrong tho